docs/containers.conf.5.md

```diff
@@ -0,0 +1,339 @@
+% containers.conf(5) Container engine configuration file
+
+# NAME
+containers.conf - The container engine configuration file specifies default
+configuration options and command-line flags for container engines.
+
+# DESCRIPTION
+Container engines like Podman & Buildah read containers.conf file, if it exists
+and modify the defaults for running containers on the host. containers.conf uses
+a TOML format that can be easily modified and versioned.
+
+Container engines read the /usr/share/containers/containers.conf and
+/etc/containers/containers.conf files if they exists.  When running in rootless
+mode, they also read $HOME/.config/containers/containers.conf files.
+
+Fields specified in containers conf override the default options, as well as
+options in previously read containers.conf files.
+
+Note not all options are supported in all container engines.
```

Comment:
I think it's worth pointing out that there are torage.conf, policy.conf and registries.conf for more specialized tweaks. Some users might be looking for such options in the containers.conf file.

Reply:
It's mentioned at the very end but I would prefer to have it here for an easier navigation.

Reply:
@rhatdan, still open. It's a non-blocking preference.

Reply:
Fixed